packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.html

```diff
@@ -48,6 +48,21 @@ <h2 data-translate="dedicatedCloud_tab_datastores_title"></h2>
             data-ng-bind="'dedicatedCloud_tab_datastores_status_' + $row.status | translate"
         ></span>
     </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'dedicatedCloud_tab_datastores_location_datacenter' | translate"
+    >
+        <span data-ng-bind="$row.location.datacenter || ''"></span>
```

Comment:
null values are not displayed if I'm not wrong

```suggestion
        <span data-ng-bind="$row.location.datacenter"></span>
```

Reply:
done, thanks